includes/User.php

```diff
@@ -4617,4 +4617,22 @@ public function jsonSerialize() {
 			'mName' => $this->mName,
 		];
 	}
+
+	public static function getIndexedUserNames( $userIds ): array {
```

Comment:
Let's mark it as Wikia's change.

Reply:
Let's use `User::whoAre` instead https://github.com/Wikia/app/pull/14034